frontend/src/containers/Ledger.js

```diff
@@ -138,6 +150,15 @@ export class Ledger extends Component {
 
         { this.props.route.type === 'expenses' && <div className='Ledger-container padding40 expenses-container'>
             <div className='line1'>unpaid expenses</div>
+
+            {isHost && 
```

Comment:
PaypalReminder needs to decide which message to show and for that it needs the card. It also needs info on details on the card - for example email and eventually a message to expire the card. 

I think it's easier to have a component that's passed a paypalCard and it decides how to display it. We will need to show it other places as well.